### DISCUSSION TITLE
Possible issue in `alias: named parameter expressions` section of `HelloMojo` notebook

### DISCUSSION THREAD

**Discussion by armgabrielyan** [modular/modular#370] [Q&A] [answered]
For this, Mojo uses an `alias` declaration. For example, the `DType` struct implements a simple enum using aliases for the
enumerators like this (the actual internal implementation details vary a bit):

```mojo
struct dtype:
    alias invalid = 0
    alias bool = 1
    alias int8 = 2
    alias uint8 = 3
    alias int16 = 4
    alias uint16 = 5
    alias float32 = 15
```

This allows clients to use `DType.float32` as a parameter expression (which also works as a runtime value of course) naturally.

`alias: named parameter expressions` section of `HelloMojo` describes the idea of named parameter expressions on the example of `DType` struct from the standard library. Also, it defines a `struct` named `dtype` and later it refers to the fact that it allows clients to use `DType.float32` as a parameter expression. In my opinion, this creates confusion because it is not obvious to me as a reader if `dtype` is a typo and should be `DType` or if it refers to the type in the standard library indeed. Also, the same section of `Programming Manual` defines struct `DType` as follows.

```mojo
struct DType:
    var value : UI8
    alias invalid = DType(0)
    alias bool = DType(1)
    alias int8 = DType(2)
    alias uint8 = DType(3)
    alias int16 = DType(4)
    alias int16 = DType(5)
    ...
    alias float32 = DType(15)
```

**Comment by mojodojodev** [comment, marked as answer]
I'm not sure if this has changed, but in the most recent version of the programming manual, there is only a capitalized `DType` with no lowercase version

**Reply by drunkwcodes** [reply]
I second that.

**Reply by armgabrielyan** [reply]
Yes, it looks good now. Thank you!
